aura_calculator.py

```
@@ -0,0 +1,149 @@
+        self.update_balance_label()
+        self.update_action_history()
+        messagebox.showinfo("Success", f"Action added: {action} ({points} points)")
+
+    def delete_action(self):
+        selected_text = self.history_listbox.get("1.0", "end").splitlines()
+        if not selected_text or selected_text[0] == "":
+            messagebox.showerror("Error", "No action selected for deletion!")
+            return
+        
+        user_data = self.users[self.current_user]
+        action_text = selected_text.pop(0)
```

Ruff

Local variable `action_text` is assigned to but never used

Remove assignment to unused variable `action_text`

(F841)